tests/unit/test_row_noise.py

```diff
@@ -114,6 +115,134 @@ def test_do_not_respond_missing_columns(dummy_data):
         _ = NOISE_TYPES.do_not_respond("silly_dataset", dummy_data, config, RANDOMNESS)
 
 
-@pytest.mark.skip(reason="TODO")
-def test_duplication():
-    pass
+def test_guardian_duplication():
+    # We are going to make a small dataframe and update the configuration to noise 100% of the
+    # available rows. We will then check that the correct rows were copied with the correct
+    # information.
+    dummy_data = pd.DataFrame(
+        {
+            "simulant_id": [str(i) for i in list(range(10))],
```

Comment:
I like what you're doing here, but can it be reduced even further? Seems like there's some duplicated simulant functionality but I can't quite sort out how many combinations of thigns actually exist

Reply:
THe duplication is to show that dependents at different addresses that copy a guardian copy the guardian's address.